include/bat/ads/ad_info.h

```diff
@@ -2,7 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#pragma once
+#ifndef BAT_NATIVE_ADS_INCLUDE_BAT_ADS_AD_INFO_H_
```

Comment:
sorry, I missed that there was also some additional incorrect path. It should always match the include path which is BAT_ADS_AD_INFO_H_

Reply:
Fixed